### PR TITLE
MBS-9422 leftovers for TMS plugin

### PR DESCRIPTION
**Pull request by dsvoronin**
- failfast at configuration time if no reportHost provided
- rewrite log messages to clearly understand state

### CODE REVIEW

File: subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt [hunk 1]
```diff
@@ -19,42 +19,46 @@ class MarkReportAsSourceAction(
...
+                    preparedData = jsonObject(
+                        "full" to true,
+                        "version" to testSuiteVersion
+                    )
+                ).fold(
+                    {
+                        reportsApi.setFinished(reportCoordinates).fold(
+                            {
+                                logger.info(
+                                    "[$tag] Test suite for tms version $testSuiteVersion, " +
```
Comment (dsvoronin) [inline subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt:42]: do we need tags like that? it's not straightforward to me how to filter build log for TMS messages only

Comment (eugene-krivobokov) [reply subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt:42]: CILogger already appends a prefix. Isn't that enough?

**Review by eugene-krivobokov — APPROVED** [review]

File: subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt [hunk 2]
```diff
@@ -19,42 +19,46 @@ class MarkReportAsSourceAction(
...
-
-    private fun tryGetId(reportCoordinates: ReportCoordinates): String? {
-        return when (val result = reportsApi.getReport(reportCoordinates)) {
-            is GetReportResult.Found -> result.report.id
             GetReportResult.NotFound -> {
-                logger.warn("Can't find report for runId=${reportCoordinates.runId}")
-                null
+                logger.critical(
+                    "[$tag] Can't get reportId for coordinates: $reportCoordinates",
+                    IllegalStateException("stub")
```
Comment (sergioboi) [inline subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt:57]: Add message

File: subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt (hunk 1 above)
Comment (dsvoronin) [reply subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt:42]: stdout destination doesn't print any tag as i see; we should handle it; but not in this PR

File: subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt (hunk 2 above)
Comment (dsvoronin) [reply subprojects/gradle/tms/src/main/kotlin/com/avito/plugin/MarkReportAsSourceAction.kt:57]: i want optional Throwable arg here, or i need stub exceptions like here

**Pull request merged**